changelogs/fragments/1355-vmaware_cluster_vcls.yaml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - vmware_cluster_vcls.py - Add the possibility to override the default vCLS VM disk placement for this cluster.
+    (https://github.com/ansible-collections/community.vmware/pull/1355)
```

Comment:
Could you please remove the changelog fragment? When creating the changelog, our tooling should automatically mention `vmware_cluster_vcls` in the `New Modules` section. Adding a minor change would be a little redundant.